pbt/__init__.py

```diff
@@ -1,8 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
 import logging
-from logging import NullHandler
+from logging.config import dictConfig as _dictConfig
+from os import path
+
+import yaml
+
+__author__ = 'Giorgio Ruffa <gioruffa@gmail.com>'
```

Comment:
Thanks for contributing to the project!

I am flattered to be included as an author, but most of the code was written by Fernando.
Consider also adding yourself to the list of contributors in the `__credit__` header field.
```suggestion
__author__ = 'Fernando Diaz <fediazgon@gmail.com>, Giorgio Ruffa <gioruffa@gmail.com>'
__credits__ = 'your name and email'
```